drand/drand_leader_msg_test.go

```diff
@@ -24,3 +24,21 @@ func TestConstructInitMessage(test *testing.T) {
 		test.Errorf("Init message is not constructed in the correct size: %d", len(msg))
 	}
 }
+
+func TestProcessCommitMessage(test *testing.T) {
+	leader := p2p.Peer{IP: "127.0.0.1", Port: "19999"}
+	validator := p2p.Peer{IP: "127.0.0.1", Port: "55555"}
+	priKey, _, _ := utils.GenKeyP2P("127.0.0.1", "9902")
+	host, err := p2pimpl.NewHost(&leader, priKey)
+	if err != nil {
+		test.Fatalf("newhost failure: %v", err)
+	}
+	dRand := New(host, "0", []p2p.Peer{leader, validator}, leader, nil, true)
+	dRand.blockHash = [32]byte{}
+	msg := dRand.constructCommitMessage([32]byte{}, []byte{})
+
+	if len(msg) != 191 {
```

Comment:
to compare the length sometimes fail in travis_checker?  do you know of anything else makes more sense to check?

Reply:
We can unmarshall the messagepayload and check for the msg type. Let me know your thoughts.

Reply:
sounds good.